### PR TITLE
Refactor/details

### LINKED ISSUE #96: Remove all Console Logs and other devtrash
- [x] remove all console logs
- [x] check all variables
- [x] :root, font-family / font-size refactoring

### PR DESCRIPTION
**Pull request by ChristianMLux**
changes some details, 
removes console logs

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -26,25 +26,27 @@ export default {
...
   --background-color: #fafafa;
   --success-color: #a3db33;
   --fail-color: #ed3221;
   --font-color: #121212;
   --placeholder-color: rgba(18, 18, 18, 0.7);
+  font-family: "Open Sans", sans-serif;
+  font-size: 18px;
+  line-height: 1.5rem;
+}
+#app {
```
Comment (ChristianMLux) [inline src/App.vue:41]: moves some styling to :root element

File: src/App.vue [hunk 2]
```diff
@@ -26,25 +26,27 @@ export default {
...
   -moz-osx-font-smoothing: grayscale;
   text-align: center;
 }
 html {
   @import url("https://fonts.googleapis.com/css2?family=Open+Sans&display=swap");
-  font-family: "Open Sans", sans-serif;
   background-color: var(--background-color);
   color: var(--font-color);
-  font-size: 18px;
-  line-height: 1.5rem;
```
Comment (ChristianMLux) [inline src/App.vue:47]: removes lines because they are not needed there

File: src/App.vue [hunk 3]
```diff
@@ -60,7 +62,7 @@ body {
   margin: 0 auto;
   min-width: 350px;
 }
-@media screen and (max-width: 600px) {
+@media screen and (max-width: 669px) {
```
Comment (ChristianMLux) [inline src/App.vue:65]: breakpoint change to 669

File: src/components/AskQuestion/AskQuestion.vue [hunk 4]
```diff
@@ -426,7 +426,7 @@ textarea {
   color: var(--font-color);
 }
 
-@media screen and (max-width: 600px) {
+@media screen and (max-width: 669px) {
```
Comment (ChristianMLux) [inline src/components/AskQuestion/AskQuestion.vue:429]: breakpoint change to 669

File: src/components/AskQuestion/RadioButton.vue [hunk 5]
```diff
@@ -232,7 +232,7 @@ input[type="radio"]:checked + .category-sonstiges-input {
   }
 }
 
-@media screen and (max-width: 600px) {
+@media screen and (max-width: 669px) {
```
Comment (ChristianMLux) [inline src/components/AskQuestion/RadioButton.vue:235]: breakpoint change to 669

File: src/components/MainButton/MainButton.vue [hunk 6]
```diff
@@ -55,7 +55,7 @@ export default {
   }
 }
 
-@media screen and (max-width: 600px) {
+@media screen and (max-width: 669px) {
```
Comment (ChristianMLux) [inline src/components/MainButton/MainButton.vue:58]: breakpoint change to 669

File: src/components/QuestionDetails/QuestionDetails.vue [hunk 7]
```diff
@@ -75,8 +75,6 @@ export default {
   display: flex;
   margin-left: 2.5rem;
 }
-.button {
-}
```
Comment (ChristianMLux) [inline src/components/QuestionDetails/QuestionDetails.vue:79]: removes unused css constructor

File: src/components/TeamMember/TeamMember.vue [hunk 8]
```diff
@@ -83,7 +83,7 @@ export default {
 .github-pic {
   height: 2rem;
 }
-@media screen and (max-width: 600px) {
+@media screen and (max-width: 669px) {
```
Comment (ChristianMLux) [inline src/components/TeamMember/TeamMember.vue:86]: breakpoint change to 669

File: src/components/UserLogin/UserLogin.vue [hunk 9]
```diff
@@ -35,7 +35,6 @@ export default {
       this.user = result.user;
       this.userID = result.user.uid;
       this.userName = result.user.displayName;
-      console.log(this.userName);
```
Comment (ChristianMLux) [inline src/components/UserLogin/UserLogin.vue:38]: removes console log

File: src/components/UserLogin/UserLogin.vue [hunk 10]
```diff
@@ -67,7 +66,6 @@ export default {
     margin-left: 0.5rem;
   }
 }
-
```
Comment (ChristianMLux) [inline src/components/UserLogin/UserLogin.vue:70]: empty line

File: src/components/UserLogin/UserLogin.vue [hunk 11]
```diff
@@ -79,7 +77,7 @@ export default {
 }
 .btn-ggl-login {
   font-size: 1.2rem;
-  color: red;
+  color: var(--fail-color);
```
Comment (ChristianMLux) [inline src/components/UserLogin/UserLogin.vue:80]: using variable

**Review by robinboehm — APPROVED** [review]

**Comment by robinboehm** [comment]
LGTM

**Pull request merged**